cloudscraper/interpreters/__init__.py

```diff
@@ -0,0 +1,115 @@
+import re
+import sys
+import logging
+import abc
+import os
+
+from io import open
+
+interpreters = dict()
+with open(os.path.join(os.path.dirname(__file__), 'interpreters.lst')) as fp:
+    interpreters.update((k, None) for k in fp.read().split('\n'))
```

Comment:
Reading the interpreters from a file to prevent having to modify this code every time we add a new one